release-notes/8.0/supported-os.json

```diff
@@ -0,0 +1,48 @@
+{
+  "channel-version": "8.0",
+  "last-updated": "2024-02-28",
+  "glibc": [
+    {
+      "name": "Ubuntu",
+      "supported-cycles": [
+        {
+          "cycle": "16.04",
+          "architectures": ["x64", "arm64", "arm"],
+          "required-version": "2.23"
+        }
+      ],
+      "last-updated": "2024-03-21"
+    }
+  ],
+  "musl": [
+    {
+      "name": "Alpine",
+      "supported-cycles": [
+        {
+          "cycle": "3.13",
+          "architectures": ["x64", "arm64", "arm"],
+          "required-version": "1.2.2"
+        }
+      ],
+      "last-updated": "2024-03-21"
+    }
+  ],
+  "operating-systems" : [
+    {
+      "name": "Ubuntu",
+      "lifecycle-policy": "https://wiki.ubuntu.com/Releases",
+      "architectures": ["x64", "arm64", "arm"],
+      "supported-cycles" : ["20.04", "22.04", "23.10", "24.04"],
+      "unsupported-cycles" : ["18.04"],
+      "last-updated": "2023-02-21"
+    },
+    {
+      "name": "iOS",
+      "lifecycle-policy": "https://support.apple.com/ios",
```

Comment:
should this maybe point to an aka.ms? In the unlikely even that URL ever changes, then we only have to update one address, e.g. aka.ms/dotnet-ios-support

Reply:
I'd prefer not.

- Transparent links are greatly preferred. The opaque links are effectively proprietary.
- It's not our responsibility to prevent broken links for third parties. If broken links appear, folks can propose fixes.
- Let's keep this "system" as lean as possible.